jobs/release.Jenkinsfile

```diff
@@ -371,6 +375,17 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
     currentBuild.result = 'FAILURE'
     throw e
 } finally {
-    def color = currentBuild.result == 'SUCCESS' ? 'good' : 'danger'
+    def color
+    switch(currentBuild.result) {
```

Comment:
I wonder if we should fold this into `trySlackSend()`. I think we want this in every call site. If we ever have a caller in the future that wants to control the colour not based on build status, it could override the color parameter.

Reply:
let me see what I can do

Reply:
added a new commit on top (and one other one too)